server/routes/viewingLicences/handlers/viewLicence.ts

```diff
@@ -69,6 +73,23 @@ export default class ViewAndPrintLicenceRoutes {
     }
   }
 
+  POST = async (req: Request, res: Response): Promise<void> => {
+    const { licenceId } = req.params
+    const { user, licence } = res.locals
+
+    const pduHeads = await this.communityService.getPduHeads(licence.probationPduCode).then(p =>
+      p.map(c => {
+        return {
+          name: `${c.staff.forenames} ${c.staff.surname}`,
+          email: c.email,
+        }
+      })
+    )
+    await this.licenceService.submitVariation(licenceId, pduHeads, user)
```

Comment:
This is for CAs creating hard-stop licences, correct?
Should we be contacting the PDU heads for these creations? PDU heads are the heads of probation, and I thought are typically only contacted to approve variations

Reply:
Shall I pass CA details instead?

Reply:
I don't think you need to submit anything
I suppose it depends what the ticket says as to whether need to be emailed, but I would assume you can just use `submitLicence` rather than `submitVariation`. They both call the same endpoint in the API, and looking at that endpoint it seems like the only time the second param (`NotifyRequest` is used is when the licence kind is `VARIATION`, so it's currently redundant anyway.

If you want to email the prison approver, it seems like you'd need to make changes to the current API endpoint regardless. I think we already send emails through some mechanism when a licence is submitted though, so it may be worth checking if just using `submitLicence` would result in an email being sent anyway, in which case you probably wouldn't need to add anything.

Reply:
that makes sense - emails is a separate ticket also 👍